### DISCUSSION TITLE
Change username/name of account

### DISCUSSION THREAD

**Discussion by Kuratar** [monkeytypegame/monkeytype#1254] [Questions] [answered]
Is there a feature to change the username or the name of the account displayed at the top? It's currently my real name and would prefer if it was a username I desired.

**Comment by unrealapex** [comment]
Monkeytype has no feature to do so yet. Did you sign in with Google or register?

**Reply by Kuratar** [reply]
Signed up normally when registering then at some point, had to log in again and used the google method second time around.

**Comment by Miodec** [comment]
Did you sign up with gmail to begin with? Or did you sign up as usual and then signed in with gmail? There is a bug currently where it overrides your name, but if you sign up with gmail it should ask you to change your name.

**Reply by Kuratar** [reply]
I think this is the case. I initially signed up with normally as in entering my email and password. At some point, I deleted my cookies so I had to sign in again and I believe I used the google method of signing in.

**Reply by unrealapex** [reply]
>There is a bug currently where it overrides your name

@Miodec Try making usernames constant so username values will not be able to be changed?, again  I am a still a newbie web developer, my expertise is mainly in  `Python` and `Java` so pardon me if I get some things wrong.

**Reply by jd0x0021** [reply]
@Miodec I signed in with google, I just changed my name in the alert box, but I can't change it back again. I hope we can change it in the settings in future updates 👨‍💻

**Comment by Brachyurus** [comment]
I have the same issue. I created my account without google, but after signing in with google once, my name has changed.

**Comment by Miodec** [comment, marked as answer]
@Kuratar @Brachyurus Get in touch with me either on Discord or email I will update your names.
